### DISCUSSION TITLE
Can Next Auth be used alongside Supabase?

### DISCUSSION THREAD

**Discussion by baptisteArno** [nextauthjs/next-auth#3043] [Help]
I have a Supabase application and I need a Spotify login. Is there a way I can fully integrate NextAuth with Supabase row policies?

**Comment by balazsorban44** [comment]
there is an intent to add a supabase adapter. see https://github.com/nextauthjs/adapters/pull/188

Could take any advise/help on it.

**Comment by thorwebdev** [comment]
UPDATE: A supabase adapter has been added to NextAuth and RLS supported has been added to the docs: https://next-auth.js.org/adapters/supabase#enabling-row-level-security-rls

**Reply by akinwol** [reply]
Is there a updated link? This one doesn't work.

**Reply by agustif** [reply]
This link to the SupabaseAdapter works https://authjs.dev/reference/adapter/supabase
